recipes/zlib/1.2.11/conanfile.py

```diff
@@ -69,7 +69,8 @@ def _build_zlib_autotools(self):
         if "clang" in str(self.settings.compiler) and tools.get_env("CC") is None and tools.get_env("CXX") is None:
             env = {"CC": "clang", "CXX": "clang++"}
         with tools.environment_append(env):
-            env_build.configure("../", build=False, host=False, target=False)
+            #env_build.configure("../", build=False, host=False, target=False)
```

Comment:
@a4z I'm curious about this change. Could you explain is it necessary?

Reply:
the arguments have no effect for the zlib build, so why add to not use them 
on the other side, it is also not wrong to add them, since they change nothing